pkg/storage/credentialset_test.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func TestNewCredentialSet(t *testing.T) {
-	cs := NewCredentialSet("dev", "mycreds", secrets.SourceMap{
+	cs := NewCredentialSet("dev", "mycreds", "", secrets.SourceMap{
```

Comment:
Wouldn't it make sense assert that the description is added correctly like this test does with the other inputs?